package.json

```diff
@@ -189,16 +188,20 @@
     "postcss-normalize": "^10.0.1",
     "prettier": "^2.6.2",
     "process": "^0.11.10",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "react-select-event": "^5.3.0",
-    "react-test-renderer": "16.14.0",
+    "react-test-renderer": "^18.2.0",
+    "rollup-plugin-polyfill-node": "^0.10.2",
     "sinon": "^7.2.2",
     "stream-browserify": "^3.0.0",
     "stream-http": "^3.2.0",
     "timers-browserify": "^2.0.12",
     "ts-jest": "^28.0.5",
     "typescript": "4.7.4",
-    "vite": "^2.9.12",
-    "vite-plugin-externals": "^0.5.0",
+    "vite": "^3.0.9",
```

Comment:
Breaking changes did not affect us, needed to upgrade to v3 to use more recent versions of @storybook/builder-vite